app/commands/tooling.go

```diff
@@ -0,0 +1,117 @@
+package commands
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/codegangsta/cli"
+	"github.com/codelingo/lingo/app/util"
+	"github.com/codelingo/lingo/service"
+	"github.com/juju/errors"
+	"strings"
+)
+
+func init() {
+	register(&cli.Command{
+		Hidden: true,
+		Name:   "tooling",
+		Usage:  "Tools",
```

Comment:
"Commands under tooling are intended to support the other software such as IDEs and are not intended to be called by the user directly from the CLI"